src/sentry/tasks/post_process.py

```diff
@@ -564,6 +445,160 @@ def process_snoozes(group):
     return False
 
 
+def process_event(cache_key, data, group_id):
+    from sentry.eventstore.models import Event
+    from sentry.models import EventDict
+
+    event = Event(
+        project_id=data["project"], event_id=data["event_id"], group_id=group_id, data=data
+    )
+
+    set_current_event_project(event.project_id)
+
+    # Re-bind node data to avoid renormalization. We only want to
+    # renormalize when loading old data from the database.
+    event.data = EventDict(event.data, skip_renormalization=True)
+
+    return event
+
+
+def process_event_group(event):
+    # NOTE: we must pass through the full Event object, and not an
+    # event_id since the Event object may not actually have been stored
+    # in the database due to sampling.
+    from sentry.models.group import get_group_with_redirect
+
+    # Re-bind Group since we're reading the Event object
+    # from cache, which may contain a stale group and project
+    event.group, _ = get_group_with_redirect(event.group_id)
+    event.group_id = event.group.id
+
+    # We fetch buffered updates to group aggregates here and populate them on the Group. This
+    # helps us avoid problems with processing group ignores and alert rules that rely on these
+    # stats.
+    with sentry_sdk.start_span(op="tasks.post_process_group.fetch_buffered_group_stats"):
+        fetch_buffered_group_stats(event.group)
+
+    event.group.project = event.project
+    event.group.project.set_cached_field_value("organization", event.project.organization)
+
+
+def process_rules(event, is_new, is_regression, is_new_group_environment, has_reappeared):
+    from sentry.rules.processor import RuleProcessor
+
+    rp = RuleProcessor(event, is_new, is_regression, is_new_group_environment, has_reappeared)
+
+    has_alert = False
+    with sentry_sdk.start_span(op="tasks.post_process_group.rule_processor_callbacks"):
+        # TODO(dcramer): ideally this would fanout, but serializing giant
+        # objects back and forth isn't super efficient
+        for callback, futures in rp.apply():
+            has_alert = True
+            safe_execute(callback, event, futures, _with_transaction=False)
+
+    return has_alert
+
+
+def process_commits(event):
```

Comment:
We never call this

Reply:
good catch x2. adding back. linter should have caught it too